lib/export.ts

```
@@ -96,6 +96,194 @@ export function blockNoteToMarkdown(content: unknown[]): string {
+export function blockNoteToLaTeX(content: unknown[]): string {
+  if (!Array.isArray(content)) {
+    return "";
+  }
+
+  let latex = "";
+
+  content.forEach((blockUnknown) => {
+    const block = blockUnknown as BlockNoteBlock;
+    switch (block.type) {
+      case "heading":
+        const level = (block.props?.level as number) || 1;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)